stumpy/stump.py

```diff
@@ -689,7 +688,7 @@ def stump(T_A, m, T_B=None, ignore_trivial=True, normalize=True, p=2.0, k=1):
     else:
         diags = np.arange(-(n_A - m + 1) + 1, n_B - m + 1, dtype=np.int64)
 
-    P, PL, PR, I, IL, IR = _stump(
+    P, _, _, I, IL, IR = _stump(
```

Comment:
I think it's okay to leave it as `PL, PR` as it will make it easier for other people to possibly use those variables in the future if necessary.

Reply:
In general, I would agree.